praw/models/reddit/emoji.py

```diff
@@ -78,6 +81,45 @@ def delete(self):
         )
         self._reddit.request("DELETE", url)
 
+    def update(
+        self,
+        post_flair=None,
+        user_flair=None,
+        mod_only=None,
```

Comment:
Could the method's parameters (and their documentation) please be alphabetized, as in [the API docs](https://www.reddit.com/dev/api#POST_api_v1_{subreddit}_emoji.json)?

Reply:
Noted and taken care of.